src/base.ts

```diff
@@ -31,6 +31,7 @@ export enum CoinKey {
   METIS = 'METIS',
   SOL = 'SOL',
   EVM = 'EVM',
+  AETH = 'AETH',
```

Comment:
So far we don't have new coin keys for every wrapped version of ETH (would be quite many). We just keep the `ETH` Coinkey and have the specific abbrevation saved as the `symbol`:
https://github.com/lifinance/types/blob/0f7eff874e7dce1446bfb0b4dacb18b91cb9475f/src/coins.ts#L583-L588
https://github.com/lifinance/types/blob/0f7eff874e7dce1446bfb0b4dacb18b91cb9475f/src/coins.ts#L792-L797

If you want to have the the coinkey specifically I think it's enough to cast it:
https://github.com/lifinance/types/blob/0f7eff874e7dce1446bfb0b4dacb18b91cb9475f/src/coins.ts#L1142-L1152

But not sure if we want it to be `WETH` as coinkey so we can map every wrapped eth version to another on all the chains